CHANGELOG.md

```diff
@@ -1,3 +1,288 @@
+# 85.0.0
+
+In this major release we updated `sizeThatFits:` calculation for MDCBottomNavigationBar to not include safe area, added Ripple support to all our interactable components, and more layout customization for Banner.
+
+## Breaking changes
+
+We have changed the `sizeThatFits:` behavior for BottomNavigation's bar to not include the safe area by default. This is a breaking change as prior we did include the safe area as part of the calculation, but we should not since the view controller (or parent view) should worry about positioning the BottomNavigationBar within the unsafe area (if it desires).
+
+If you still wish to include the safe area in the sizeThatFits calculation:
+```objc
+self.bottomNavigationBar.sizeThatFitsIncludesSafeArea = YES;
+```
+
+## New features
+
+Enable the new Ripple behavior in components by setting `enableRippleBehavior` to YES, and easily set its color using the `rippleColor` API:
```

Comment:
Delete `easily`; is generally wise not to include subjective language in our documentation.